code/game/objects/structures/window.dm

```diff
@@ -310,6 +310,17 @@
 	max_integrity = 300
 	reinf = TRUE
 
+//For the sulaco and POS AI core.
+/obj/structure/window/reinforced/extratoughened
+	name = "protective AI glass"
+	desc = "Heavily reinforced glass with many layers of a rod matrice. This is rarely used for anything but the most important windows"
+	icon_state = "rwindow"
+	basestate = "rwindow"
+	max_integrity = 1500
+	reinf = TRUE
+	resistance_flags = 10 // I have no clue what those are.
```

Comment:
unacidable and xeno_damageable i suppose ?

Reply:
I do not know, that's what the hard coded stuff had. You would have to ask the other person what those do.